src/cpr_data_access/models.py

```diff
@@ -759,7 +759,8 @@ def _load(
 
         parser_outputs = adaptor.load_dataset(name_or_path, limit)
         self.documents = [
-            self.document_model.from_parser_output(doc) for doc in parser_outputs
+            self.document_model.from_parser_output(doc)
+            for doc in tqdm(parser_outputs, desc="Parsing documents")
```

Comment:
i think desc here should be _"Loading documents"_ or similar as we use the term 'parsing' for 'extracting text from' in other places

Reply:
I've changed the wording both here and at the local file reading to make it clear 🙌